### PR TITLE
Update homepage and remove mention of Bourbon

### PR DESCRIPTION
**Pull request by whmii**
- https://github.com/thoughtbot/neat/issues/449
- https://github.com/thoughtbot/neat/issues/337

screencapture-localhost-4000-1467826279932

### CODE REVIEW

**Comment by whmii** [comment]
A quick side note here. Some of these stylesheets are a little crazy so I made some choices I wouldn't normally do. I think it is probably better to make as few changes as possible here.

**Comment by tysongach** [comment]
Looks good.

Is this a duplicate of https://github.com/thoughtbot/neat/pull/452 or does it close it?

File: _sass/desktop.scss [hunk 1]
```diff
@@ -42,14 +42,14 @@ section.code {
 }
 
 section.instructions {
-  section {
+  > section {
```
Comment (jasonramirez) [inline _sass/desktop.scss]: ich. Do we have to? Can we add class names to the sections? This feels like a move to a sunset.

**Comment by jasonramirez** [comment]
@whmii made a comment about some ug css. Seems to hit on:

> A quick side note here. Some of these stylesheets are a little crazy so I made some choices I wouldn't normally do. I think it is probably better to make as few changes as possible here.

Feel free to ignore if you don't think its going to improve things going forward.

🍔  away!

**Comment by whmii** [comment]
@tysongach this is an alternate to #452 

I wanted to hit a few issues in one PR

**Comment by dijonkitchen** [comment]
FYI, Bourbon is still mentioned in the logo and url.

**Comment by tysongach** [comment]
> FYI, Bourbon is still mentioned in the logo and url.

@dijonkitchen Neat is still part of the Bourbon suite, it just no longer requires it as a dependency.

**Pull request merged**